testing/trino-testing-containers/src/main/java/io/trino/testing/minio/MinioClient.java

```diff
@@ -227,6 +230,26 @@ public void copyObject(String sourceBucket, String sourceKey, String targetBucke
         }
     }
 
+    public void removeObjects(String bucket, String path)
```

Comment:
This method seems to be unused. Is it a left-over?

Reply:
Thanks for catching this. Removed the left-over.